apps/meteor/app/api/server/v1/banners.ts

```diff
@@ -52,9 +52,8 @@ import { API } from '../api';
  */
 API.v1.addRoute(
 	'banners.getNew',
-	{ authRequired: true },
```

Comment:
Just curious, is that the the real alternative? I see the endpoint returning one or all banners 👀 but the new one apparently just returns one.

If that's the case, we should put the endpoint to return more than one as an alternative

Reply:
Oops, you're right. bid can be undefined
I'll add `banners` as an alternative too